_1327/information_pages/templates/information_pages_base.html

```diff
@@ -10,24 +10,28 @@
 {% endblock %}
 
 {% block sidebar %}
-	{{ toc|safe }}
 	<dl class="meta-information">
 		<dt>{% trans "Author" %}</dt>
 		<dd>{{ document.author }}</dd>
 		<dt>{% trans "Created" %}</dt>
-		<dd>{{ document.created }}</dd>
+		<dd>{{ document.created|date:"d. M Y, H:i" }}</dd>
```

Comment:
I mean you're explicitly setting the date format. But ``d. M Y`` is not used everywhere.